modules/ws-client/src/main/scala/trading/client/ui/AlertsUI.scala

```diff
@@ -0,0 +1,36 @@
+package trading.client
+package ui
+
+import cats.syntax.show.*
+
+import tyrian.{ Attr, Html }
+import tyrian.Html.*
+
+private def mkAlert(property: Option[String], divId: String, status: String, message: String): Html[Msg] =
+  //FIXME: div [ hidden property.isEmpty ]
+  val alertHidden: Attr[Nothing] =
+    if property.isEmpty then hidden else attribute("foo", "")
```

Comment:
@davesmith00000 do you know if there's a better way to do this? Is there an "empty" HTML element? In Elm, the `hidden` property takes a `Boolean` as an argument to decide whether it's present or not. In Scala, I had to get around by adding a dummy attribute when I don't want the `hidden` element.

```elm
div [ hidden (M.isEmpty property), id divId ]
```

Reply:
I think there isn't currently, but that's a good idea. Have raised an issue: https://github.com/PurpleKingdomGames/tyrian/issues/54

Reply:
Actually there is an empty attribute:
```scala
object Attribute:
  val empty: Attribute = Attribute("", "")
```